### PR TITLE
Enhanced RDoc for Integer

### PR DESCRIPTION
**Pull request by BurdetteLamar**
Treats:
- #<<
- #>>
- #to_s

### CODE REVIEW

File: numeric.c [hunk 1]
```diff
@@ -4777,12 +4777,20 @@ int_xor(VALUE x, VALUE y)
 }
 
 /*
- * Document-method: Integer#<<
- * call-seq:
- *   int << count  ->  integer
+ *  call-seq:
+ *    int << count -> integer
```
Comment (jeremyevans) [inline numeric.c:4781]: Don't we typically use `self` as the receiver in this case, instead of `int`?  Same issue with `>>`.

**Review by jeremyevans — APPROVED** [review]
Looks good.  I think it might be good to make call-seq more consistent by using `self`, as noted inline, but I'll leave that up to you.

File: numeric.c (hunk 1 above)
Comment (BurdetteLamar) [reply numeric.c]: There were three of these, now fixed.

**Comment by BurdetteLamar** [comment]
I noticed in the html that there were several call-seqs cited with (p1).  This meant that the doc for the method is not correctly placed (should immediately precede the code).  These are now fixed.
Finally, there was another (p1) for try_convert, b/c it had no doc at all!  Will push a fix for that separately.

**Review by jeremyevans — APPROVED** [review]
Looks good!

**Pull request merged**